test/e2e/suite/command/inspect.go

```diff
@@ -34,7 +34,7 @@ var (
 		"├── user defined attributes",
 		"│   └── (empty)",
 		"├── unsigned attributes",
-		"│   └── signingAgent: Notation/",
+		"│   └── signingAgent: notation-go/",
```

Comment:
Why are we updating `signingAgent` here? Notation looks correct to me.

Reply:
This is a change due to notation-go [signingAgent](https://github.com/notaryproject/notation-go/blob/84c2ec076201697ad05e5315444812f824dad56b/signer/signer.go#L37). If we keep `Notation` here, the e2e test would fail.